algorithms.tex

```diff
@@ -1,2 +1,17 @@
+% A description of some of the algorithms in SymPy. The list is not
```

Comment:
I too think it makes more sense to mention the algorithms in the respective feature. Since most of our algorithms are from literature, we can describe modifications/optimizations that we did (if any).